05_syntax_lab/03.py

```diff
@@ -1,7 +1,19 @@
-"""
-Write a program that randomizes a number
-and prints the sum total of its digits.
-For example if the number was: 2345
-The result should be: 14
-"""
-
+"""
+      _      _      _
+   __(.)< __(.)> __(.)=
+   \___)  \___)  \___)   
+
+Python course - Verint
+Week 1
+
+"""
+from random import randint

```

Comment:
👍 
1. do leave some spaces for readability for example `sum += random % 10`
2. it'd be nice to print the number before the loop so we can see the calculation worked